blogposts/announcing-sourcegraph-3.6.md

```diff
@@ -0,0 +1,162 @@
+</p>
+-->
+
+[Quick links](https://docs.sourcegraph.com/user/quick_links) can now be added below the search bar on the home page and search results page. This makes it easier for teams to quickly access the Sourcegraph pages they care most about such as your team’s main repository, or have easy access to external resources like your CI system or team documentation.
+
+Add a quick link to global, organization, or user settings with the `quicklinks` field:
+
+```
+{
+  // ...
+  "quicklinks": [
+    {
+      "name": "Sourcegraph docs",
+      "url": "https://docs.sourcegraph.com"
+    },
+    {
+      "name": "Main repository",
+      "url": "/github.com/sourcegraph/sourcegraph"
+    }
+  ]
+  // ...
+}
+```
+
+## Added support for Bitbucket Cloud external service configuration
+
+TO DO: VIDEO
+
+<!--
+<p class="container">
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://player.vimeo.com/video/{ID}?color=0CB6F4&amp;title=0&amp;byline=" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://vimeo.com/{ID}" target="_blank">View on Vimeo</a></p>
+</p>
+-->
+
+Previously, Bitbucket Cloud repositories had to be added and configured as individual git repositories. In Sourcegraph 3.6 we added first class support for connecting [Bitbucket Cloud repositories](https://docs.sourcegraph.com/admin/external_service/bitbucket_cloud). In this release, all repositories will be added for the username configured, and teams listed that the Bitbucket Cloud app password grants access to.
+
+## Improved external service configuration and syncing experience for admins
+
+TO DO: Updated video (should we mark the 3.5 video as deprecated?)
+
+<!--
+<p class="container">
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://player.vimeo.com/video/{ID}?color=0CB6F4&amp;title=0&amp;byline=" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://vimeo.com/{ID}" target="_blank">View on Vimeo</a></p>
+</p>
+-->
+
+Admins no longer have to wait for an external service to finish syncing before navigating away from the configuration page. Previously, this was problematic for customers adding external services with lots of repositories. Now, the syncing continues in the background and progress is shown to admins in the global navigation bar’s syncing status indicator.
+
+The experimental repository syncing status indicator announced in 3.5 has been improved enough to turn this feature on for all site admins by default. Improvements include handling scale of 30,000+ repositories, and only showing the number of repositories remaining to be cloned.
```

Comment:
I don't think
```suggestion
This progress status UI has been improved to support instances with 30,000+ repositories, only showing the number of repositories remaining to be cloned.
```

Reply:
Let me know about the change I made as an alternative.